pulsar-websocket/src/main/java/org/apache/pulsar/websocket/data/ProducerMessage.java

```diff
@@ -69,4 +70,13 @@ public class ProducerMessage {
 
     // Base64 encoded serialized schema for payload
     public String valueSchema;
+
+    // Compression type. Do not set it if compression is not performed.
+    public String encryptionParam;
+
+    // The size of the payload before compression. Do not set it if compression is not performed.
+    public CompressionType compressionType;
+
+    // Base64 encoded serialized initialization vector used when the client encrypts.
```

Comment:
It's an integer, shouldn't be base64 encoded?

Reply:
Fixed